doc/source/computation.rst

```diff
@@ -196,90 +196,118 @@ parameter:
+computing common *window* or *rolling* statistics. Among these are count, sum,
 mean, median, correlation, variance, covariance, standard deviation, skewness,
-and kurtosis. All of these methods are in the :mod:`pandas` namespace, but
-otherwise they can be found in :mod:`pandas.stats.moments`.
+and kurtosis.
 
-.. currentmodule:: pandas
+.. currentmodule:: pandas.core.window
 
-.. csv-table::
-    :header: "Function", "Description"
-    :widths: 20, 80
+.. note::
 
-    :func:`rolling_count`, Number of non-null observations
-    :func:`rolling_sum`, Sum of values
-    :func:`rolling_mean`, Mean of values
-    :func:`rolling_median`, Arithmetic median of values
-    :func:`rolling_min`, Minimum
-    :func:`rolling_max`, Maximum
-    :func:`rolling_std`, Unbiased standard deviation
-    :func:`rolling_var`, Unbiased variance
-    :func:`rolling_skew`, Unbiased skewness (3rd moment)
-    :func:`rolling_kurt`, Unbiased kurtosis (4th moment)
-    :func:`rolling_quantile`, Sample quantile (value at %)
-    :func:`rolling_apply`, Generic apply
-    :func:`rolling_cov`, Unbiased covariance (binary)
-    :func:`rolling_corr`, Correlation (binary)
-    :func:`rolling_window`, Moving window function
-
-Generally these methods all have the same interface. The binary operators
-(e.g. :func:`rolling_corr`) take two Series or DataFrames. Otherwise, they all
+   The API for window statistics is quite similar to the way one works with ``Groupby`` objects, see the documentation :ref:`here <groupby>`
+
+We work with ``rolling``, ``expanding`` and ``exponentially weighted`` data through the corresponding
+objects, :class:`~pandas.core.window.Rolling`, :class:`~pandas.core.window.Expanding` and :class:`~pandas.core.window.EWM`.
+
+.. ipython:: python
+
+   s = pd.Series(np.random.randn(1000), index=pd.date_range('1/1/2000', periods=1000))
+   s = s.cumsum()
+   s
+
+These are created from methods on ``Series`` and ``DataFrames``.
+
+.. ipython:: python
+
+   r = s.rolling(window=60)
+   r
+
+Generally these methods all have the same interface. They all
 accept the following arguments:
 
-  - ``window``: size of moving window
-  - ``min_periods``: threshold of non-null data points to require (otherwise
-    result is NA)
-  - ``freq``: optionally specify a :ref:`frequency string <timeseries.alias>`
-    or :ref:`DateOffset <timeseries.offsets>` to pre-conform the data to.
-    Note that prior to pandas v0.8.0, a keyword argument ``time_rule`` was used
-    instead of ``freq`` that referred to the legacy time rule constants
-  - ``how``: optionally specify method for down or re-sampling.  Default is
-    is min for :func:`rolling_min`, max for :func:`rolling_max`, median for
-    :func:`rolling_median`, and mean for all other rolling functions.  See
-    :meth:`DataFrame.resample`'s how argument for more information.
+- ``window``: size of moving window
+- ``min_periods``: threshold of non-null data points to require (otherwise
```

Comment:
Sorry if this is in the wrong place but I couldnt find a better place to ask this question:
When will pandas 0.18 be available? reason I am asking I think I may have found a bug with the rolling_min function, when applied on a forward looking basis. Since the window functions seem to be completely overhauled in the new release, I would want to retest it in the new version before reporting the bug.